### DISCUSSION TITLE
Clarification about script flags

### DISCUSSION THREAD

**Discussion by microraptor** [007revad/Synology_HDD_db#134] [Q&A] [answered]
I am planning to make a storage volume on two Samsung SSDs in the M.2 slots of a DS423+ with a third-part RAM module. So I assume the recommended flags are `-nr` and then schedule the script for every boot.

I am really not sure about the `-n` and `-f` flags. Are they recommended to use or do they lead to more potential issues than they solve? In what cases are they useful?

Thank you for all the work you put into the scripts :)

**Comment by 007revad** [comment, marked as answer]
Definitely use the -n flag. I would use -nr in your case.

The -f is more brute force and rarely needed.

**Reply by microraptor** [reply]
Thank you for your 5 min fast tech support!

**Comment by 007revad** [comment]
PS the -r flag is only needed if you have more RAM than officially supported.
